config.md

```diff
@@ -308,15 +308,26 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value MUST be strings.
+While the key and value MUST be present, they MAY be empty strings.
```

Comment:
Empty-string values are fine, but empty-string keys sounds like a bad idea to me.